README.md

```diff
@@ -68,13 +77,13 @@ In the `main` branch:
 
 #### Submitting a PR to multiple release branches
 
-Ideally all changes will go to `main` and then be promoted to a release branch. However, ocassionally we will need to fix documentation issues for already released versions. For such cases, a PR must be created for each release branch.
+Ideally all changes will go to `main` and then be promoted to a release branch. However, occasionally we will need to fix documentation issues for already released versions. For such cases, a PR must be created for each release branch. Rendered policies will always go to all branches because the policy samples themselves declare minimum capable versions via the `policies.kyverno.io/minversion` annotation.
```

Comment:
> Rendered policies will always go to all branches because the policy samples themselves declare minimum capable versions via the `policies.kyverno.io/minversion` annotation.

Not sure about this, as there may be policies that depend on new features not available in older releases. Why would we want to put those in prior release? 

I think in general we should only merge / cherry-pick fixes, or other critical items, but otherwise always commit to main and release as a new patch, etc.

Reply:
It's a matter of consistency but also may serve as motivation for upgrading. The policies themselves declare the minimum version, so that serves as the gate for users. As a result, we simplify if we just push all policy rendered MDs to all branches and let the filter assist users in locating their policies. Not doing that means complicating the process to render all policies, delete ones matching a certain query for that `minversion` annotation (which would be manual), and pushing the rest. Cherry picking one commit to X number of branches is simpler and more consistent.

Reply:
We can try it out, but I prefer minimizing changes to released versions as a best practice.

Reply:
What's your concern here specifically? The website has filters in place to provide similar levels of convenience that the whole docs versioning drop-down was meant to provide. Plus, as we roll forward there are going to be policies contributed which *don't* rely upon the latest release and so those still need to be made available. Any policy rendering can be stored as its own commit making it easy to identify which docs changes are modified content and which are just static material like sample policies.

Reply:
The concern is creating extra work for committers, and potential errors and confusion for users.

As mentioned, its possible a policy is not supported in a prior version. Or, perhaps it does not work the same way. Or, there may be bugs. Sure, the version is specified in the policies but it also seems easy to miss when trying out a sample. And unless we test a new policy, and any changes, in each prior version, it does not seem like a good idea to have a blanket guideline to always render all policies in each version.

Also policies may need to change across versions. Again, just rendering and committing to all branches will cause problems. We want to preserve what was previously tested and known to work against the prior release, not overwrite it.

There may be some case we want to add sample policies changes to prior releases (e.g. bug fixes), but a more prudent default would be to focus on "main" which is tested and released with as a unit along with the software.

Reply:
Maybe we huddle on this in Slack for a few minutes to discuss at some point. I think one thing we could do is create a new annotation that would indicate the version of Kyverno against which the policy was developed. As far as testing of these policies against each release, this is ultimately where we want to go (provided the CLI supports those policies) and are taking steps to add tests for the policies. In fact, one of my next initiatives (after getting the 1.4.2 docs up to speed) is to finish building out all those test resources. We should then plan to integrate them into our e2e tests to ensure this isn't an issue as we roll forward with releases.